src/components/AgentChats.vue

```diff
@@ -20,6 +20,22 @@
             <p>3</p>
           </span>
         </section>
+        <section class="item table-row">
+          <span class="table-col agent">
+            <p>Fabricio Correia</p>
+          </span>
+          <span class="table-col" style="text-align: center">
+            <p>3</p>
+          </span>
+        </section>
+        <section class="item table-row">
+          <span class="table-col agent">
+            <p>Fabricio Correia</p>
+          </span>
+          <span class="table-col" style="text-align: center">
+            <p>3</p>
+          </span>
+        </section>
```

Comment:
Adjust semantics, element classes using _BEM_ and remove the inline style (choose to insert a class with the required style)